hw_8/index.js

```diff
@@ -0,0 +1,48 @@
+window.addEventListener('DOMContentLoaded', (event) => {
+    const userBYear = prompt('Введіть Ваш рік народження.', '');  
+    const userCity = prompt('В якому місті Ви проживаєте?', '');
+    const userSport = prompt('Введіть Ваш улюблений вид спорту.', '');
+
+    const today = new Date().getFullYear();
+    const userAge = today - +userBYear;
```

Comment:
It's okay but as far as you decided to convert to a number here -- you forgot about NaN. and `userAge`could be NaN

Reply:
thank you for the comment, fixed this issue
can you check it, please

Reply:
https://github.com/anastasia1383/VasylievaFrontEndPro251022/pull/6/commits/17fab78c46f0f440a627db78260b50d04f1f75b8